example/1_getstarted.html

```diff
@@ -20,6 +20,8 @@
 </div>
 
 <script src="../lib/three.min.js"></script>
+<script src="../lib/three.eventdispatcher.js"></script>
+

```

Comment:
could we just use `THREE.EventDispatcher.prototype` instead loading external lib?

Reply:
https://github.com/mrdoob/three.js/blob/dev/build/three.module.js
https://github.com/mrdoob/eventdispatcher.js/blob/master/src/EventDispatcher.js

three.module.js has THREE.EventDispatcher with it, it was taken out of three.core.js when it was depreciated in 74  according to the documentation. It was then moved to its own repository/modules which is the only one I added, the CDN has the module available but I included it from the original repository.